### ISSUE TITLE
Plays in background when pressing media key

### ISSUE THREAD

**Issue by ErnestoArtigas** [corbindavenport/wii-shop-extension#25]
First of all, this extension is a great idea and it's so much emotional to hear this beautiful track when shopping around !

However, I don't know how, but when pressing my physical media key (on a Logitech keyboard) it plays the Wii Shop Theme, even though I don't have any e-commerce websites in the background. Is there any way to disable media control keys for the extension, to avoid this issue ?

**Comment by corbindavenport** [comment]
Yeah I've noticed this, it's a side result of using [a standard HTML5 audio object](https://github.com/corbindavenport/wii-shop-extension/blob/da0573c36f9303acbb6bb3ba9010dee9e311a936/js/background.js#L13) in the background page for media playback. Chromium browsers show it in the media control toolbar button, and it can be controlled with media keys, same as videos and music in web pages.

The only way around this that I know of would be to [switch to the Web Audio API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API), but that's more complicated and I haven't had time to through it. That might also be required when the extension gets migrated to Manifest V3 (#7), since background pages work differently, but I'm not sure yet.

**Comment by ErnestoArtigas** [comment]
If you want to get a simplier Web Audio / HTML5 API you can use HowlerJS, I've used it many times in different projects and is very to use ! https://github.com/goldfire/howler.js
